index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
```

Comment:
- [ ] You have done an excellent job with your project and it's looking great :fire: but one of the requirements it's to add a descriptive README and it's recommended to use [this README template](https://github.com/microverseinc/readme-template) and customize it for your project :wink:
![image](https://user-images.githubusercontent.com/81522435/143049865-f0ff32af-c874-4e4a-a630-3610dd84c9b2.png)